lib/samples/authenticate_with_oauth/authenticate_with_oauth.dart

```diff
@@ -23,40 +23,34 @@ class AuthenticateWithOAuth extends StatefulWidget {
   const AuthenticateWithOAuth({super.key});
 
   @override
-  State<AuthenticateWithOAuth> createState() =>
-      _AuthenticateWithOAuthState();
+  State<AuthenticateWithOAuth> createState() => _AuthenticateWithOAuthState();
 }
 
-class _AuthenticateWithOAuthState
-    extends State<AuthenticateWithOAuth>
+class _AuthenticateWithOAuthState extends State<AuthenticateWithOAuth>
     with SampleStateSupport
     implements ArcGISAuthenticationChallengeHandler {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // Create an OAuthUserConfiguration.
   // This document describes the steps to configure OAuth for your app:
-  // https://developers.arcgis.com/documentation/mapping-apis-and-services/security/user-authentication/serverless-native-flow/
+  // https://developers.arcgis.com/documentation/security-and-authentication/user-authentication/flows/authorization-code-with-pkce/
```

Comment:
This comment block also appears in `show_portal_user_info.dart`. Could you make the same changes over there?